src/components/CodeOverview/index.spec.tsx

```diff
@@ -201,35 +201,6 @@ describe(__filename, () => {
     expect(root.state('overviewHeight')).toEqual(fakeRef.current.clientHeight);
   });
 
-  it('only sets the overview height for defined refs', () => {
```

Comment:
Why is this patch removing these tests?

Reply:
While looking at this code (for unrelated reasons), I realized that these tests check cases that are already covered by TypeScript so they are pointless [1]. 

The first test covers what TypeScript would cover here:

```
src/components/CodeOverview/index.tsx:85:37 - error TS2531: Object is possibly 'null'.

85     this.setState({ overviewHeight: ref.current.clientHeight });
                                       ~~~
```

And the second one:

```
src/components/CodeOverview/index.tsx:86:39 - error TS2531: Object is possibly 'null'.

86       this.setState({ overviewHeight: ref.current.clientHeight });
                                         ~~~~~~~~~~~
```

[1] One could say that the point of the tests is to boost our coverage score but we don't actually count *if* branches so they don't affect our score anyway.